test/PushNotificationTests/APITests.cpp

```diff
@@ -77,150 +99,96 @@ namespace Test::PushNotifications
         TEST_METHOD_SETUP(MethodInit)
         {
             VERIFY_IS_TRUE(TP::IsPackageRegistered_ProjectReunionFramework());
-
-            m_failed = CreateTestEvent(c_testFailureEventName);
-
             return true;
         }
 
         TEST_METHOD_CLEANUP(MethodUninit)
         {
             VERIFY_IS_TRUE(TP::IsPackageRegistered_ProjectReunionFramework());
 
+            if (m_process)
+            {
+                VERIFY_IS_TRUE(CloseHandle(m_process));
+            }
             return true;
         }
 
-        TEST_METHOD(ChannelRequestUsingNullRemoteId)
+        void RunTest(const PCWSTR& testName)
         {
-            wil::unique_event event = CreateTestEvent(c_testProtocolScheme_Packaged);
+            DWORD pid;
+            VERIFY_SUCCEEDED(m_testAppLauncher.get()->ActivateApplication(L"PushNotificationsTestAppPackage_8wekyb3d8bbwe!App", testName, AO_NONE, &pid));
+            VERIFY_IS_NOT_NULL(pid);
```

Comment:
There is a better TAEF macro.

```suggestion
            VERIFY_ARE_NOT_EQUAL(processId);
```

Reply:
Actually double check if ActivateApplication can return success and processId == 0. If not, then you don't need the check.